src/implementations/vanilla/components/form/_form.scss

```diff
@@ -3,19 +3,10 @@
  */
 
 @use 'sass:map';
-@use '@ecl/theme-dev/theme';
-
```

Comment:
i wish we could be able to provide a form-group template, one of the most questionable things in ECL about the twig templates is the duplication in all the form elements of the related code. This is how is done in webcomponents, for instance, but in twig it would be different. Still we could handle all the rendering in the form group template and use the single form element templates only to pass params to it. The form group would then have dozens of params, probably, but that complexity can be completely handled on our side, keeping the same "experience" in the implementation for the users..

Reply:
this could be a good opportunity to do it, indeed.
But I guess that we should discuss it with EWPP, to see how we could provide usable template. Form elements are usually quite tricky to handle

Reply:
I created the ticket https://citnet.tech.ec.europa.eu/CITnet/jira/browse/FRONT-3995 to see how to handle that